scripts/TetToQuadConnection/quadMeshingFunctions.py

```diff
@@ -0,0 +1,110 @@
+# A python script meant to run the process of taking a tet mesh and creating a quad mesh of one of the boundaries.
```

Comment:
Move this file into the scripts/ folder, and let's get rid of the TetToQuadConnection folder, since this has been slimmed down.